PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+## Description
+
+Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.
+
+Fixes # (issue)
+
+## Type of change
+
+Please delete options that are not relevant.
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
+- [ ] Documentation update
+
+## Checklist:
+
+- [ ] My code follows the [style guidelines](CONTRIBUTING.md) of this project
```

Comment:
Update this link to point to the corresponding section in our org's contributing guidelines (see last comment above)